build.gradle

```diff
@@ -2,24 +2,41 @@ plugins {
     id 'org.jetbrains.kotlin.jvm' version "$kotlinVersion"
     id 'org.jetbrains.kotlin.plugin.serialization' version "$kotlinVersion"
     id 'org.openapi.generator' version "$openApiGenVersion"
+    id 'com.github.johnrengelman.shadow' version "$shadowJarVersion"
+    id 'de.undercouch.download' version "$downloadPluginVersion"
     id 'java'
     id 'application'
     id 'idea'
 }
 
-// Allow custom specification of Cineast OpenAPI config via -PapiConfigPath=...
-def apiConfigPath = project.properties["apiConfigPath"] ?: "$rootDir/cineast.json"
+// OpenAPI specifications (VREM & Cineast).
+def specPath = "$rootDir/oas"
+
+// Path for generated Cineast client sources.
+def generatedPath = "$buildDir/generated"
+
+// Path for Cineast OpenAPI specification.
+def cineastOasPath = "$specPath/cineast-oas.json"
+
+// VREM OpenAPI specification URL and local file.
+def vremOasUrl = "http://localhost:4545/swagger-docs"
+def vremOasPath = "$specPath/vrem-oas.json"
 
 // Source/Target compatibility (currently used for source & target/JVM).
 def javaCompatibility = 11
 
 // Project settings.
 group 'ch.unibas.dmi.dbis'
-version '2.1.0'
+version '3.0.0-SNAPSHOT'
 
 // Main class.
 mainClassName = 'ch.unibas.dmi.dbis.vrem.VREMKt'
 
+// Only build the shadow jar and corresponding distributions.
+jar.enabled = false
+tasks.distZip.enabled = false
+tasks.distTar.enabled = false
```

Comment:
Same as comment before: why disabling the building of distributions?

Reply:
`./gradlew build` already creates all the distributions via shadowJar task, so they will still be available in the `build/distributions` folder - I thought I'd exclude the regular application distZip/Tar/jar.
I'm not entirely sure if this has any negative impact, but that is the case the changes should of course be reversed.